src/components/core/accordion/accordion.tsx

```diff
@@ -0,0 +1,173 @@
+    is_expanded: boolean;
+    size: 'medium' | 'small';
+    type: 'bottomBorder' | 'container' | 'containerWithBorder' | 'containerWithShadow';
+};
+
+const AccordionDiv = styled('div', {
+    variants: {
+        elevation_type: {
+            xs: {
+                boxShadow: 'none',
+            },
+            sm: {
+                boxShadow: 'none',
+            },
+            md: {
+                boxShadow: 'none',
+            },
+            lg: {
+                boxShadow: 'none',
+            },
+            xl: {
+                boxShadow: 'none',
+            },
+            xxl: {
+                boxShadow: 'none',
+            },
+            xxxl: {
+                boxShadow: 'none',
+            },
+        },
+        type: {
+            bottomBorder: {
+                borderBottom: '1px solid $greyLight300',
+            },
+            container: {
+                borderRadius: '24px',
+            },
+            containerWithBorder: {
+                borderRadius: '8px',
+                border: '1px solid $greyLight300',
+            },
+            containerWithShadow: {
+                boxShadow: '$xs',
+            },
+        },
+        dark: {
+            true: {
+                borderColor: '$greyDark500',
+            },
+        },
+    },
+    compoundVariants: [
+        {
+            type: 'bottomBorder',
+            dark: true,
+            css: {
+                borderColor: '$greyDark500',
+            },
+        },
+        {
+            type: 'containerWithBorder',
+            dark: true,
+            css: {
+                borderColor: '$greyDark500',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'xs',
+            css: {
+                boxShadow: '$xs',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'sm',
+            css: {
+                boxShadow: '$sm',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'md',
+            css: {
+                boxShadow: '$md',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'lg',
+            css: {
+                boxShadow: '$lg',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'xl',
+            css: {
+                boxShadow: '$xl',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'xxl',
+            css: {
+                boxShadow: '$xxl',
+            },
+        },
+        {
+            type: 'containerWithShadow',
+            elevation_type: 'xxxl',
+            css: {
+                boxShadow: '$xxxl',
+            },
+        },
+    ],
+});
+
+const Accordion = ({
+    title_content,
```

Comment:
`title_content` and `main_content` should be from `Accordion.Title` and `Accordion.Content` respectively.
We should not be passing the title and content as a prop.

Reply:
This is how we use the Wizard Component outside of `@deriv/ui`
<img width="737" alt="image" src="https://user-images.githubusercontent.com/56330681/183010300-94722ff7-701f-40c9-bc29-9a5b4d7edede.png">

In the same way, Accordion should be used like

<Accordion ...>
    <Accordion.Title ...>
    </Accordion.Title>
    <Accordion.Content ...>
    </Accordion.Content>
</Accordion>